config/locales/en.yml

```diff
@@ -376,6 +376,8 @@ en:
         update_failure: "Could not save changes. Error: %{error_msg}"
         delete: "Delete order type"
         confirm_delete: "There are %{count} listings with this order type. If you delete the order type, those listings will be closed. Are you sure you want to delete the order type?"
+        can_not_delete_last: "You can't delete this order type because it's the only one type in your marketplace."
```

Comment:
"You can't delete this order type because it's the only one type in your marketplace."
->
"You can't delete this order type because it's the only type in your marketplace."